apps/web/src/app/vault/individual-vault/view.component.ts

```diff
@@ -0,0 +1,166 @@
+import { DIALOG_DATA, DialogConfig, DialogRef } from "@angular/cdk/dialog";
+import { CommonModule } from "@angular/common";
+import { Component, Inject, OnInit, EventEmitter } from "@angular/core";
+import { Router } from "@angular/router";
+
+import { OrganizationService } from "@bitwarden/common/admin-console/abstractions/organization/organization.service.abstraction";
+import { Organization } from "@bitwarden/common/admin-console/models/domain/organization";
+import { I18nService } from "@bitwarden/common/platform/abstractions/i18n.service";
+import { LogService } from "@bitwarden/common/platform/abstractions/log.service";
+import { MessagingService } from "@bitwarden/common/platform/abstractions/messaging.service";
+import { CipherService } from "@bitwarden/common/vault/abstractions/cipher.service";
+import { CipherType } from "@bitwarden/common/vault/enums/cipher-type";
+import { CipherView } from "@bitwarden/common/vault/models/view/cipher.view";
+import {
+  AsyncActionsModule,
+  DialogModule,
+  DialogService,
+  ToastService,
+} from "@bitwarden/components";
+
+import { CipherViewComponent } from "../../../../../../libs/vault/src/cipher-view/cipher-view.component";
+import { SharedModule } from "../../shared/shared.module";
+
+export interface ViewCipherDialogParams {
+  cipher: CipherView;
+  cipherTypeString: string;
+}
+
+export enum ViewCipherDialogResult {
+  edited = "edited",
+  deleted = "deleted",
+}
+
+export interface ViewCipherDialogCloseResult {
+  action: ViewCipherDialogResult;
+}
+
+@Component({
+  selector: "app-vault-view",
+  templateUrl: "view.component.html",
+  standalone: true,
+  imports: [CipherViewComponent, CommonModule, AsyncActionsModule, DialogModule, SharedModule],
+})
+export class ViewComponent implements OnInit {
+  cipher: CipherView;
+  deletePromise: Promise<void>;
+  onDeletedCipher = new EventEmitter<CipherView>();
+  cipherTypeString: string;
+  cipherEditUrl: string;
+  organization: Organization;
+  flexibleCollectionsV1Enabled = false;
+  restrictProviderAccess = false;
+
+  constructor(
+    @Inject(DIALOG_DATA) public params: ViewCipherDialogParams,
+    private dialogRef: DialogRef<ViewCipherDialogCloseResult>,
+    private i18nService: I18nService,
+    private dialogService: DialogService,
+    private messagingService: MessagingService,
+    private logService: LogService,
+    private cipherService: CipherService,
+    private toastService: ToastService,
+    private organizationService: OrganizationService,
+    private router: Router,
+  ) {}
+
+  async ngOnInit() {
+    this.cipher = this.params.cipher;
+    this.cipherTypeString = this.params.cipherTypeString;
+    if (this.cipher.organizationId) {
+      this.organization = await this.organizationService.get(this.cipher.organizationId);
+    }
+  }
+
+  async delete(): Promise<boolean> {
```

Comment:
This is an old pattern that I find myself following as well. Or copying and pasting on accident 🙃 

The `boolean` return isn't used for anything and can be removed

```suggestion
  async delete(): Promise<void> {
```